test/functional/test_pipemode.py

```diff
@@ -126,8 +126,7 @@ def input_fn():
         ds = ds.repeat(count=2)
         ds = ds.prefetch(3)
         ds = ds.batch(3)
-        it = ds.make_one_shot_iterator()
```

Comment:
Why do we remove this?

Reply:
it was removed in most other places during the [upgrade to 2.0](https://github.com/aws/sagemaker-tensorflow-extensions/commit/65293dd3908107d5aba5c1419cc9226182d0f09d) - not entirely sure why this one got missed